Core/DetectorConstruction.cpp

```diff
@@ -590,6 +609,18 @@ void DetectorConstruction::ConstructTargetRun3()
     transform, unionSolid_logical, "bigChamberInnerStructure",
     fWorldLogical, true, 0, checkOverlaps
   );
+  
+  // Vacuum
+  G4Tubs* bigChamberRun3_vac = new G4Tubs(
+    "bigChamberRun3_vac", 0 * cm, chamber_radius_inner - 0.01 * cm,
+    31. * cm , 0 * degree, 360 * degree);
```

Comment:
Can you add a comment in code that 31 cm is a length of the inner chamber, so in future it would be clear for developers?

Reply:
or maybe as in constructTargetRun5 define a variable, lets say 
G4double vacuumChamber_halfLength = 31*cm;
? and use it here instead of 31. * cm
You can also comment why did you use 'chamber_radius_inner - 0.01 * cm'

Reply:
Correction:
const double vacuumChamber_halfLength = 31*cm;